src/lib/components/empty.svelte

```diff
@@ -23,17 +24,19 @@
     <article class="card u-grid u-cross-center u-width-full-line common-section">
         <div
             class="u-flex u-flex-vertical u-cross-center u-gap-24 u-width-full-line u-overflow-hidden">
-            <button
-                type="button"
-                on:click|preventDefault
-                on:click={track}
-                aria-label="create {target}">
-                {#if $app.themeInUse === 'dark'}
-                    <img src={EmptyDark} alt="create" aria-hidden="true" width="376" />
-                {:else}
-                    <img src={EmptyLight} alt="create" aria-hidden="true" width="376" />
-                {/if}
-            </button>
+            {#if !noMedia}
```

Comment:
This can be confusing, a potential improvement could be having a `hasMedia` variable instead

Reply:
> This can be confusing, a potential improvement could be having a `hasMedia` variable instead

this is the only case without showing a media - so the exception. Otherwise it would need to have `hasMedia` in 10+ places 👍🏻

Reply:
I see!